hibernate-reactive-core/src/main/java/org/hibernate/reactive/mutiny/Mutiny.java

```diff
@@ -1621,6 +1621,21 @@
 		 */
 		<T> Uni<T> withTransaction(BiFunction<Session, Transaction, Uni<T>> work);
 
+		/**
+		 * Perform work using a {@link StatelessSession reactive session} within an
+		 * associated {@link Transaction transaction}.
+		 * <p>
+		 * The session will be {@link Session#flush() flushed} and closed
+		 * automatically, and the transaction committed automatically.
+		 *
+		 * @param work a function which accepts the session and returns
+		 *             the result of the work as a {@link Uni}.
+		 *
+		 * @see #withSession(Function)
+		 * @see StatelessSession#withTransaction(Function)
+		 */
```

Comment:
@blafond this comment needs to change a bit, because there is no notion of flushing for a stateless session. Also the @see annotations point to the wrong things.

Reply:
working on it

Reply:
comments fixed